JiraPS/Private/Resolve-ErrorWebResponse.ps1

```diff
@@ -44,7 +44,29 @@ function Resolve-ErrorWebResponse {
             try {
                 $responseObject = ConvertFrom-Json -InputObject $responseBody -ErrorAction Stop
 
+                $errorArray = @()
                 foreach ($_error in ($responseObject.errorMessages + $responseObject.errors)) {
+
+                    foreach ($_err in $_error.PSObject.Properties.Value) {
+                        if ($_err -like "*You must specify a summary of the issue." -Or $_err -like "*is required.") {
+                            Write-Debug $_err
+                            $errorArray += $_err
+                        } else {
+                            # unhandled value
+                        }
+                    }
+
+                    if ($errorArray.count -gt 0) {
+                        $errorParameter = @{
+                            ExceptionType = "System.Net.Http.HttpRequestException"
+                            Message       = "Missing Required Fields: $errorArray"
+                            ErrorId       = "AuthenticationFailed"
+                            Category      = "AuthenticationError"
+                            Cmdlet        = $Cmdlet
+                        }
+                        ThrowError @errorParameter
```

Comment:
You shouldn't use `throw` here. `Throw` will cause the entire script to fail when this happens (unless used with `try {} catch {}`.
Please use `WriteError` (can have the same behavior if wanted, by adding `-ErrorAction Stop` to the script)